interface/main/calendar/add_edit_event.php

```diff
@@ -116,9 +126,16 @@
     $collectthis = $collectthis[array_keys($collectthis)[0]]["rules"];
 }
 ?>
+<?php $disabled = (!$edit && $GLOBALS['enable_group_therapy'] )?' disabled=true; ':'';?>
+<?php if($disabled) echo '<script>$( document ).ready(function(){
```

Comment:
This looks like you are disabling the event editor if group therapy is enabled and the user does not have gcalendar permission. But what if a practice wants to offer group therapy as well as standard (non-group) visits? Is that not supported?